src/main/java/nl/avans/drivio/service/UserService.java

```diff
@@ -0,0 +1,36 @@
+package nl.avans.drivio.service;
+
+
+import nl.avans.drivio.model.User;
+import nl.avans.drivio.repository.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class UserService {
+
+    private final UserRepository userRepository;
+
+    @Autowired
+    public UserService(UserRepository userRepository) {
+        this.userRepository = userRepository;
+    }
+
+    public List<User> getAllUsers() {
+        return (List<User>) userRepository.findAll();
+    }
+
+    public Optional<User> getById(int id) {
+        return userRepository.findById(id);
+    }
+
+    public void add(User user) {
+        userRepository.save(user);
+    }
+
+    public void delete(User user) {
+        userRepository.delete(user);
+    }
+}
```

Comment:
De UserService laag wordt nog nergens gebruikt. In de controller verwijs je nog naar de repository. Deze moet nog ingebouwd worden.

Reply:
Wordt gefixed in issue #29